pygen/pygen_src/riscv_instr_gen_config.py

```diff
@@ -329,36 +322,26 @@ def get_invalid_priv_lvl_csr(self):
         # Debug CSRs are inaccessible from all but Debug Mode
         # and we cannot boot into Debug Mode.
         invalid_lvl.append('D')
-
-        # TODO Need to change the logic once the constraints are up.
-        for mode in self.init_privileged_mode:
-            if mode == privileged_mode_t.MACHINE_MODE:
-                continue
-            if mode == privileged_mode_t.SUPERVISOR_MODE:
-                invalid_lvl.append('M')
-                logging.info("supr_mode---")
-                logging.debug(invalid_lvl)
-            elif mode == privileged_mode_t.USER_MODE:
-                invalid_lvl.append('S')
-                invalid_lvl.append('M')
-                logging.info("usr_mode---")
-                logging.debug(invalid_lvl)
-            else:
-                logging.critical("Unsupported initialization privilege mode")
+        if self.init_privileged_mode == privileged_mode_t.MACHINE_MODE:
+            pass
+        elif self.init_privileged_mode == privileged_mode_t.SUPERVISOR_MODE:
+            invalid_lvl.append('M')
+            logging.info("supr_mode---")
```

Comment:
nit: better to be consistent in using `''` vs. `""`: `invalid_lvl.append("M")`. Here as well as few other places below.